js/src/annotations/osd-svg-ellipse.js

```diff
@@ -0,0 +1,154 @@
+          overlay.path.segments[i].point.y += event.delta.y;
+        }
+      } else if (overlay.mode == 'create' || overlay.mode == 'deform') {
+        var idx = -1;
+        for (var l = 0; l < overlay.path.segments.length; l++) {
+          if (overlay.path.segments[l] == overlay.segment) {
+            idx = l;
+            break;
+          }
+        }
+        if (overlay.mode == 'deform' && idx % 2 == 1) {
+          var oldPoint = new overlay.paperScope.Point(overlay.segment.point.x - overlay.path.position.x, overlay.segment.point.y - overlay.path.position.y);
+          var newPoint = new overlay.paperScope.Point(overlay.segment.point.x - overlay.path.position.x + event.delta.x, overlay.segment.point.y - overlay.path.position.y + event.delta.y);
+          var scale = Math.sqrt(newPoint.x * newPoint.x + newPoint.y * newPoint.y) / Math.sqrt(oldPoint.x * oldPoint.x + oldPoint.y * oldPoint.y);
+          var rotation = Math.atan2(newPoint.y, newPoint.x) - Math.atan2(oldPoint.y, oldPoint.x);
+          rotation = rotation * (180 / Math.PI);
+          overlay.path.scale(scale);
+          overlay.path.rotate(rotation, overlay.path.position);
+          overlay.path.data.rotation += rotation;
+        } else {
+          var oldRotPoint = new overlay.paperScope.Point(overlay.segment.point.x - overlay.path.position.x, overlay.segment.point.y - overlay.path.position.y);
+          var newRotPoint = new overlay.paperScope.Point(overlay.segment.point.x - overlay.path.position.x + event.delta.x, overlay.segment.point.y - overlay.path.position.y + event.delta.y);
+          var rot = overlay.path.data.rotation;
+          oldRotPoint = oldRotPoint.rotate(-rot);
+          newRotPoint = newRotPoint.rotate(-rot);
+          var rotScale = new overlay.paperScope.Point(newRotPoint.x / oldRotPoint.x, newRotPoint.y / oldRotPoint.y);
+          overlay.path.rotate(-rot, overlay.path.position);
+          overlay.path.scale(rotScale);
+          overlay.path.rotate(rot, overlay.path.position);
+        }
+      }
+    },
+
+    onMouseMove: function(event, overlay) {
+      // Empty block.
+    },
+
+    onMouseDown: function(event, overlay) {
+      var hitResult = overlay.paperScope.project.hitTest(event.point, overlay.hitOptions);
+      if (hitResult && hitResult.item._name.toString().indexOf(this.idPrefix) != -1) {
+        if (overlay.mode != 'deform' && overlay.mode != 'translate' && overlay.mode != 'create') {
+          if (hitResult.type == 'segment') {
+            overlay.mode = 'deform';
+            overlay.segment = null;
+            overlay.path = null;
+            var segments = hitResult.item.segments;
+            for (var idx = 0; idx < segments.length; idx++) {
+              if (segments[idx] == hitResult.segment) {
+                if (idx % 2 === 0) {
+                  document.body.style.cursor = "move";
+                } else {
+                  document.body.style.cursor = "url('../images/rotate.png') 12 12, auto";
```

Comment:
Can you change this from the PNG file to the font awesome rotate icon? http://fortawesome.github.io/Font-Awesome/icon/repeat/